src/planwise/client/projects/views.cljs

```diff
@@ -80,6 +89,9 @@
          selected-tab)
         [:div
          [sidebar-section selected-tab]
+         [:div.loading-indicator {:style {:display (if (= @map-state :loading-displayed) "block" "none")}}
+          [:div.loading-wheel (loading-wheel)]
```

Comment:
Ok since `loading-wheel` is a constant. Otherwise it's advisable to return the function unevaluated as a component. Ie.

```clojure
[loading-wheel]
```
and move the surrounding `div` to the component itself.